Packs/ThreatConnect/Integrations/ThreatConnect_v2/ThreatConnect_v2_description.md

```diff
@@ -0,0 +1,2 @@
+## ThreatConnect v2
+Threat intelligence platform.
```

Comment:
this should contain information about "how to get the access ID and secret key"
this content of this file is presented in demisto as the help for instance configuration (question mark)
![image](https://user-images.githubusercontent.com/30797606/82752160-c6a3dd00-9dc4-11ea-96a3-e1961b24ea11.png)